packages/intrepid2/src/Orientation/Intrepid2_OrientationToolsDefModifyPoints.hpp

```diff
@@ -371,33 +371,33 @@ namespace Intrepid2 {
 
 
 
-    template<typename pointType>
+    template<typename DeviceType, typename pointType>
     KOKKOS_INLINE_FUNCTION
     void
     OrientationTools::
     getJacobianDetOfOrientationMap(pointType* jacobianDet,
                            const unsigned cellTopoKey,
                            const ordinal_type cellOrt) {
-      Kokkos::View<pointType**> jacobian;
+      Kokkos::View<pointType**,DeviceType> jacobian;
       switch (cellTopoKey) {
       case shards::Line<>::key :
       {
-        jacobian = Kokkos::View<pointType**>(jacobianDet,1,1);
+        jacobian = Kokkos::View<pointType**,DeviceType>(jacobianDet,1,1);
         getLineJacobian(jacobian, cellOrt);
         break;
       }
       case shards::Triangle<>::key :
       {
           pointType jac[4] = {};
-          jacobian = Kokkos::View<pointType**>(jac,2,2);
+          jacobian = Kokkos::View<pointType**,DeviceType>(jac,2,2);
```

Comment:
here, you wrap the jac array that is definitely host array. it is meaningless wrap with template argument of device type as it must be host memory space.